cmd/helm/flags.go

```diff
@@ -112,30 +115,70 @@ func (o *outputValue) Set(s string) error {
 }
 
 func bindPostRenderFlag(cmd *cobra.Command, varRef *postrender.PostRenderer) {
-	cmd.Flags().Var(&postRenderer{varRef}, postRenderFlag, "the path to an executable to be used for post rendering. If it exists in $PATH, the binary will be used, otherwise it will try to look for the executable at the given path")
+	p := &postRendererOptions{varRef, "", []string{}}
+	cmd.Flags().Var(&postRendererExecFlag{p}, postRenderFlag, "the path to an executable to be used for post rendering. If it exists in $PATH, the binary will be used, otherwise it will try to look for the executable at the given path")
+	cmd.Flags().Var(&postRendererArgsFlag{p}, postRenderArgsFlag, "the args to an executable to be used for post rendering.  (can specify multiple)")
+}
+
+type postRendererOptions struct {
+	renderer   *postrender.PostRenderer
+	binaryPath string
+	args       []string
 }
 
-type postRenderer struct {
-	renderer *postrender.PostRenderer
+type postRendererExecFlag struct {
+	options *postRendererOptions
 }
 
-func (p postRenderer) String() string {
-	return "exec"
+func (p postRendererExecFlag) String() string {
+	return ""
 }
 
-func (p postRenderer) Type() string {
-	return "postrenderer"
+func (p postRendererExecFlag) Type() string {
+	return "postrenderer-exec"
 }
 
-func (p postRenderer) Set(s string) error {
+func (p postRendererExecFlag) Set(s string) error {
 	if s == "" {
 		return nil
 	}
-	pr, err := postrender.NewExec(s)
+	p.options.binaryPath = s
+	pr, err := postrender.NewExecWithArgs(p.options.binaryPath, p.options.args)
 	if err != nil {
 		return err
 	}
-	*p.renderer = pr
+	*p.options.renderer = pr
+	return nil
+}
+
+type postRendererArgsFlag struct {
+	options *postRendererOptions
+}
+
+func (p postRendererArgsFlag) String() string {
+	return ""
+}
+
+func (p postRendererArgsFlag) Type() string {
```

Comment:
Since this flag can be repeated, it would be nice to let the shell completion logic know that it can suggest the flag more than once.  To achieve this you must include the string "Array" or "Slice" in the `Type()`.  You can see where the Cobra underlying library checks this: https://github.com/spf13/cobra/blob/cb9d7b1cec87c2bb005c6e2790553bcd629bc542/completions.go#L329

So can you `return "postrenderer-argsSlice"` here?

If you want to test the difference you can build your new helm and do the following:
```
$ helm __complete install --post-renderer echo --post-renderer-args arg1 --p
```
This command returns the possible completions for the specified input.  You will want to see that the `--post-renderer-args` keeps being suggested as a completion (but not the `--post-renderer` flag)